test/functionalTest/cases/2986_apiv2_notifications/apiv2_notification_reception.test

```diff
@@ -0,0 +1,304 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Reception of APIv2 notifications
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create subscription for accumulator
```

Comment:
Step 1 and 3 are not related with federation itself... Maybe they were include to guide the implementation? But now that implementation is finished, I think they shbould be removed.

Or maybe I'm not getting the point...

Reply:
Yeah, I used the first three steps to "make it work" and then I added federation.
Left the initial part in the test, but, as you say, it can be removed now, no longer necessary.

Reply:
Fixed in e56e601